step-templates/iis-bindings-restore.json

```
@@ -1,11 +1,11 @@
 {
-  "Id": "9d357886-1896-4240-a89f-118c7ca8ec05",
+  "Id": "bf7bc5d6-7576-43e6-8306-5e9e15cbab6a",

```

From the Step Template Checklist:

NOTE If you are modifying an existing step template, please make sure that you do not modify the Id property (updating the Id will break the Library sync functionality in Octopus).

```suggestion
  "Id": "9d357886-1896-4240-a89f-118c7ca8ec05",

```